valiu-node-commons/src/distribution_strategy.rs

```diff
@@ -0,0 +1,33 @@
+use core::fmt;
+
+#[cfg_attr(feature = "serde", derive(serde::Deserialize, serde::Serialize))]
+#[derive(
+    Clone,
+    Copy,
+    Debug,
+    Eq,
+    Ord,
+    PartialEq,
+    PartialOrd,
+    parity_scale_codec::Decode,
+    parity_scale_codec::Encode,
+)]
+pub enum DistributionStrategy {
+    Evenly,
+}
```

Comment:
It is possible to split collaterals by many ways and this is just one of them

Reply:
Yeah that's nice! :D that's how I thought it could be done 👍🏽